src/server/project.ts

```diff
@@ -90,6 +90,18 @@ namespace ts.server {
         }
     }
 
+    export interface PluginCreateInfo {

```

Comment:
I'm not sure if this is still a priority, but should plugins be given the opportunity to be well-behaved with respect to a cancellation token?